deploy/resources/v0.11.1/addons/pipelines/deployment-config/s2i-dotnet-3/s2i-dotnet-3-build-deploy.yaml

```
@@ -0,0 +1,46 @@
+---
+apiVersion: tekton.dev/v1alpha1
```

Can you change the apiVersion to v1beta1 in this PR itself
```suggestion
apiVersion: tekton.dev/v1beta1
```

Because for others it has been handled in [PR](https://github.com/openshift/tektoncd-pipeline-operator/pull/289)